a2sv_community_portal_mobile/lib/features/application_page/presentation/widgets/custom_card.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:a2sv_community_portal_mobile/core/utils/media_query.dart';
+import 'package:flutter/material.dart';
+
+class CustomCard extends StatelessWidget {
+  final String title;
+  final String description;
+  final bool done;
+
+  const CustomCard({
+    Key? key,
+    required this.title,
+    required this.description,
+    required this.done,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    double horPadding = UIConverter.getComponentHeight(context, 15);
+    return Container(
+      width: UIConverter.getDeviceWidth(context) >= 600
+          ? UIConverter.getComponentWidth(context, 150)
+          : null,
+      margin: EdgeInsets.only(
+        left: UIConverter.getComponentWidth(context, 27),
+        top: UIConverter.getComponentHeight(context, 41),
+        right: UIConverter.getComponentWidth(context, 27),
+      ),
+      decoration: BoxDecoration(
+        color: Colors.white,
+        borderRadius: BorderRadius.circular(8),
+        boxShadow: const [
+          BoxShadow(
+            color: Color.fromRGBO(0, 0, 0, 0.06),
```

Comment:
if you put this inside colors it would be nicer

Reply:
done